src/netspresso_trainer/dataloaders/augmentation/custom.py

```diff
@@ -349,6 +351,172 @@ def __repr__(self):
         return format_string
 
 
+class RandomMixup:
```

Comment:
Class attribute `visualize` is needed.
If it is not supporting the debug with visualizer, set the value as `False`.